package.json

```diff
@@ -25,7 +25,7 @@
     "scripts": {
         "build": "npx rollup --config",
         "watch": "npm run build -- --watch",
-        "test": "npx jest",
+        "test": "npx jest --force-exit --detectOpenHandles",
```

Comment:
`--detectOpenHandles` should help you find the non-closing process. It might also remove the message

`--force-exit` will make sure to kill the process and allow your test to run smoothly on CI environments. It is a bit barbaric but it gets the job done.